src/Administration/Resources/app/administration/src/app/component/components.js

```diff
@@ -5,6 +5,34 @@
 // eslint-disable-next-line sw-deprecation-rules/private-feature-declarations
 export default () => {
     const context = import.meta.glob('./**/index!(*.spec).{j,t}s');
+    console.error('COMPONENTS CONTEXT', context);
```

Comment:
in this file, instead of actually reading the files, we just get theis locations and invoke registration as Async components, so basically all are still registered, but loaded are only the ones which are used